### PR TITLE
catch HTTPError, not Unauthorized

### PR DESCRIPTION
**Pull request by nickpell**
https://sentry.io/dimagi/commcarehq/issues/605989965/?environment=__all_environments__

```active_tasks()``` already raises an ```HTTPError```, not ```Unauthorized```

### CODE REVIEW

**Review by emord — APPROVED** [review]
nice import cleanup

**Review by pr33thi — COMMENTED** [review]
@emord what's the resolution on my comment?

File: corehq/apps/hqadmin/views/system.py [hunk 1]
```diff
@@ -105,8 +92,11 @@ def system_ajax(request):
     if type == "_active_tasks":
         try:
             tasks = [x for x in db.server.active_tasks() if x['type'] == "indexer"]
-        except Unauthorized:
-            return json_response({'error': "Unable to access CouchDB Tasks (unauthorized)."}, status_code=500)
+        except HTTPError as e:
+            if e.response.status_code == 403:
+                return json_response({'error': "Unable to access CouchDB Tasks (unauthorized)."}, status_code=500)
```
[the remark this answers is not in the document]
Comment (emord) [reply corehq/apps/hqadmin/views/system.py]: Previously it was setting this to 500 once returned, so I think that returning 500 is just keeping the same behavior

[the remark this answers is not in the document]
Comment (pr33thi) [reply corehq/apps/hqadmin/views/system.py]: cool

**Pull request merged**